_partials/_cloud-installation.md

```diff
@@ -1,21 +1,73 @@
-## Create a Timescale Cloud account
+## Create a $CLOUD_LONG account
 
-You create a $CLOUD_LONG account to manage your services and data in a centralized and efficient manner. From $CONSOLE you can create and delete services, run queries, manage access and billing, integrate other services, contact support, and more.
+You create a $CLOUD_LONG account to manage your $SERVICE_SHORT and data in a centralized and efficient manner in $CONSOLE. From $CONSOLE_SHORT, you can create and delete $SERVICE_SHORTs, run queries, manage access and billing, integrate other services, contact support, and more.
+
+<Tabs label="Create a Timescale Cloud account">
+
+<Tab title="Standalone">
+
+You create a standalone account to manage $CLOUD_LONG as a separate unit in your infrastructure, which includes separate billing and invoicing. 
+
+To create an account:
 
 <Procedure>
 
-To set up Timescale:
+1. **Click `Try for free` on [timescale.com][timescale-website]**
 
-1.  Sign up for a [30 day free trial][sign-up].
+1. **Sign up for a 30-day free trial** 
 
     You receive a confirmation email in your inbox.
```

Comment:
Open [Sign up for Timescale][trial-signup] and add your details, then click `Sign up for Timescale`.


[trial-signup]: https://console.cloud.timescale.com/signup